apps/site/lib/site_web/views/date_time_selector.ex

```diff
@@ -0,0 +1,156 @@
+defmodule DateTimeSelector do
+  @moduledoc "Reusable customizable selector for date and time"
+  use SiteWeb, :view
+  alias Phoenix.{HTML.Form}
+  alias SiteWeb.PartialView.SvgIconWithCircle
+  import Phoenix.HTML.Tag
+  import Schedules.Repo, only: [end_of_rating: 0]
+
+  @minute_options 0..55
+                  |> Enum.filter(&(Integer.mod(&1, 5) == 0))
+                  |> Enum.map(fn
+                    int when int < 10 -> "0" <> Integer.to_string(int)
+                    int -> Integer.to_string(int)
+                  end)
+
+  def custom_date_time_select(form, %DateTime{} = datetime \\ Util.now()) do
```

Comment:
Is there a test or two we could write for this? Even something as simple as, "it takes a form and returns a div with the correct class" or something along those lines would at least be a first-line guard against regression.